src/lv_api_map_v8.h

```diff
@@ -225,6 +229,9 @@ static inline void lv_obj_move_background(lv_obj_t * obj)
 #define lv_obj_get_style_shadow_ofs_x         lv_obj_get_style_shadow_offset_x
 #define lv_obj_get_style_shadow_ofs_y         lv_obj_get_style_shadow_offset_y
 #define lv_obj_get_style_transform_angle      lv_obj_get_style_transform_rotation
+#define lv_obj_get_style_bg_img_src           lv_obj_get_style_bg_image_src
+#define lv_obj_get_style_bg_img_recolor       lv_obj_get_style_bg_image_recolor
+#define lv_obj_get_style_bg_img_recolor_opa           lv_obj_get_style_bg_image_recolor_opa
```

Comment:
```suggestion
#define lv_obj_get_style_bg_img_src           lv_obj_get_style_bg_image_src
#define lv_obj_get_style_bg_img_recolor       lv_obj_get_style_bg_image_recolor
#define lv_obj_get_style_bg_img_recolor_opa   lv_obj_get_style_bg_image_recolor_opa
```

Shouldn't be aligned?

Reply:
Thanks you, fixed!